packages/knip/src/plugins/netlify/index.ts

```diff
@@ -0,0 +1,57 @@
+import { join } from '../../util/path.js';
+import { timerify } from '../../util/Performance.js';
+import { hasDependency, load } from '../../util/plugin.js';
+import { toProductionEntryPattern } from '../../util/protocols.js';
+import { extractFunctionsConfigProperty, validFunctionExtensions } from './helpers.js';
+import type { NetlifyConfig } from './types.js';
+import type { IsPluginEnabledCallback, GenericPluginCallback } from '../../types/plugins.js';
+
+// https://docs.netlify.com/ or https://docs.netlify.com/functions/get-started/?fn-language=ts
+
+const NAME = 'Netlify';
+
+const ENABLERS = [/^@netlify\//, 'netlify-cli'];
+
+const isEnabled: IsPluginEnabledCallback = ({ dependencies }) => hasDependency(dependencies, ENABLERS);
+
+const CONFIG_FILE_PATTERNS: string[] = ['netlify.toml'];
+
+const ENTRY_FILE_PATTERNS: string[] = [];
+
+const PRODUCTION_ENTRY_FILE_PATTERNS: string[] = [`netlify/functions/**/*.{${validFunctionExtensions()}}`];
```

Comment:
Maybe we could list them explicitly here? Easier to read/update, Netlifiy does not necessarily follow Knip (or vice-versa). Also, looking at https://docs.netlify.com/functions/get-started/?fn-language=ts#module-format I only see `.mts`, `.cts` and `.ts`?

Reply:
Go to https://docs.netlify.com/functions/get-started/?fn-language=js#module-format-2. Notice how the URL now has `fn-language=js` instead of `fn-language=ts`!

Reply:
Ohh I missed that, thanks for clarifying.